### PR TITLE
resolves #1387 Core: add Alcmeon connector

### LINKED ISSUE #1387: [Core] Add Alcmeon connector
Add Alcmeon connector in order to allow integration to whatsapp or messenger with subbot mode
Functionnal overview : https://developers.alcmeon.com/docs/inmessage-functional-overview#bots
API reference : https://developers.alcmeon.com/reference/get_description-1

### PR DESCRIPTION
**Pull request by gsoumier**

### CODE REVIEW

File: bot/storage-mongo/target/generated-sources/kapt/compile/ai/tock/bot/connector/ConnectorType_.kt [hunk 1]
```diff
@@ -14,6 +16,8 @@ private val __Id: KProperty1<ConnectorType, String?>
     get() = ConnectorType::id
 private val __UserInterfaceType: KProperty1<ConnectorType, UserInterfaceType?>
     get() = ConnectorType::userInterfaceType
+private val __HandleConnector: KProperty1<ConnectorType, Function1<ConnectorType, Boolean>?>
```
Comment (gsoumier) [inline bot/storage-mongo/target/generated-sources/kapt/compile/ai/tock/bot/connector/ConnectorType_.kt:19]: ne devrait pas être serializé en base

File: pom.xml [hunk 2]
```diff
@@ -26,6 +26,14 @@
     <description>Tock, The Open Conversation Kit</description>
     <url>https://theopenconversationkit.github.io/tock</url>
 
+
+    <repositories>
```
Comment (gsoumier) [inline pom.xml:30]: à revert ?

File: bot/connector-alcmeon/src/main/kotlin/AlcmeonConnectorProvider.kt [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                    backends = parameters[BACKENDS_FIELD]?.split(",")?.map { it.trim() } ?: listOf(
+                        "whatsapp",
+                        "facebook"
+                    ),
+                    exits = listOf(
+                        AlcmeonSubBotExit(
+                            parameters[EXIT_NAME_FIELD] ?: "alcmeon",
+                            parameters[EXIT_DESCRIPTION_FIELD] ?: "Back to alcmeon customer service",
+                        )
+                    )),
```
Comment (gsoumier) [inline bot/connector-alcmeon/src/main/kotlin/AlcmeonConnectorProvider.kt:60]: May be easier to only fill one field description in configuration with the complete description

Comment (vsct-jburet) [reply bot/connector-alcmeon/src/main/kotlin/AlcmeonConnectorProvider.kt:60]: LGTM

File: bot/connector-alcmeon/src/main/resources/META-INF/services/ai.tock.bot.connector.ConnectorProvider [hunk 4]
```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2017/2021 e-voyageurs technologies
```
Comment (vsct-jburet) [inline bot/connector-alcmeon/src/main/resources/META-INF/services/ai.tock.bot.connector.ConnectorProvider:2]: normal?

File: pom.xml (hunk 2 above)
Comment (vsct-jburet) [reply pom.xml:30]: hmmm. We should use the local git repository version jar https://github.com/theopenconversationkit/tock/tree/master/nlp/entity-evaluator/duckling/service/lib

Please remove this repo :)

**Review by vsct-jburet — APPROVED** [review]

**Pull request merged**